.github/workflows/main.yml

```diff
@@ -0,0 +1,94 @@
+    - name: configure .scons_cache and download cache
+      id: scons-cache
+      uses: RevoluPowered/cache@v2.1
+      with:
+        path: ${{env.SCONS_CACHE}}
+        key: ${{ runner.os }}-godot-release_debug_editor
+        restore-keys: ${{ runner.os }}-godot-release_debug_editor
+    # use python 3.x release (works cross platform)
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        # Semantic version range syntax or exact version of a Python version
+        python-version: '3.x' 
+        # Optional - x64 or x86 architecture, defaults to x64
+        architecture: 'x64' 
+    # You can test your matrix by printing the current Python version
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+    - name: Installing scons and pywin32
+      run: |
+        python -m pip install scons pywin32
+    - name: Check versioning
+      run: |
+        python --version
+        scons --version
+    - name: Compilation
+      run: |
+        scons -j2 verbose=yes warnings=all werror=yes platform=windows target=release_debug
+    - name: publishing godot windows-editor
+      uses: actions/upload-artifact@v1
+      with:
+        name: windows-editor (x64)
+        path: bin/godot.windows.opt.tools.64.exe
+  windows-runtime:
+    runs-on: "windows-latest"
+    name: Windows Runtime (release)
+    steps:
+    - uses: actions/checkout@v2 
+    # cache must be put in the github workspace directory on windows to be persisted correctly
+    - name: "force cache path to be correct"
+      run: echo "::set-env name=SCONS_CACHE::${{env.HOME}}/.scons_cache/"
+    # upload cache on completion and check it out now
+    - name: configure .scons_cache and download cache
+      id: scons-cache
+      uses: RevoluPowered/cache@v2.1
+      with:
+        path: ${{env.SCONS_CACHE}}
+        key: ${{ runner.os }}-godot-release_runtime
+        restore-keys: ${{ runner.os }}-godot-release_runtime
+    # use python 3.x release (works cross platform)
+    - name: Set up Python 3.x
+      uses: actions/setup-python@v2
+      with:
+        # Semantic version range syntax or exact version of a Python version
+        python-version: '3.x' 
+        # Optional - x64 or x86 architecture, defaults to x64
+        architecture: 'x64' 
+    # You can test your matrix by printing the current Python version
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+    - name: Installing scons and pywin32
+      run: |
+        python -m pip install scons pywin32
+    - name: Check versioning
+      run: |
+        python --version
+        scons --version
+    - name: Compilation
+      run: |
+        scons -j2 verbose=yes warnings=all werror=yes platform=windows target=release tools=no
+    - name: publishing godot windows-editor
+      uses: actions/upload-artifact@v1
+      with:
+        name: windows-runtime-release (x64)
```

Comment:
Is this the name of the resulting download? If so, can we make it more unique by including the commit hash (shortened to 7 or 9 chars)?

Reply:
yeah i can do this

Reply:
Corrected and disabled as discussed in IRC